content/docs/sidebar.json

```
@@ -308,5 +308,76 @@
         "source": "docs.md"
       }
     ]
+  },
+  {
+    "slug": "gto",
+    "label": "GTO documentation",
```

💅🏼 I think we can just call it GTO (like https://dvc.org/doc/dvclive).

```suggestion
    "slug": "gto",
```

---

It's because we have "MLEM Documentation" on the top in mlem.ai/doc

---

I see. Same with DVC and DVClive but the nav only has "Home" and "DVCLive", respectively (it's clear that we're in a documentation site). No big deal tho